language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/FieldAccessCompletionResolver.java

```diff
@@ -82,14 +83,12 @@ public FieldAccessCompletionResolver(PositionedOperationContext context) {
 
     @Override
     public Optional<TypeSymbol> transform(SimpleNameReferenceNode node) {
-        Optional<Symbol> symbol = this.getSymbolByName(context.visibleSymbols(context.getCursorPosition()),
-                node.name().text());
-
-        if (symbol.isEmpty()) {
-            return Optional.empty();
-        }
```

Comment:
Any reason for removing this and going along with the node?

Reply:
Since the same logic was duplicated in `SimpleNameReferenceNode` and `ErrorConstructorExpressionNode`, removed this to a separate method to refactor the code.

Reply:
For the simple name reference, let's keep it as it is without changing the previous implementation. And this should be the ideal usage, but there can be edge cases where the parser/ compiler disregard the node. WDYT?

Reply:
Will do 👍